### ISSUE TITLE
runtime xunit wrapper build failed

### ISSUE THREAD

**Issue by ViktorHofer** [dotnet/runtime#45378]
```
/__w/1/s/dotnet.sh: line 27: 10092 Segmentation fault      (core dumped) ${dotnetPath} "$@"
##[error]Error: XUnit wrapper build failed. Refer to the build log files for details (above)
##[error]Bash exited with code '139'.
```

https://dev.azure.com/dnceng/public/_build/results?buildId=903083&view=logs&j=2296eecc-f5ab-523f-c15d-f45eef1bfdea&t=e1baf605-103b-50ce-aa5c-c257b320b3ad

cc @trylek @jkotas

**Comment by trylek** [comment]
@ViktorHofer - hmm, I am somewhat confused, I may be blind but I only see one failure in some iOS Mono run in that AzDO pipeline run, can it be the case that you accidentally shared the link to the wrong run?

**Comment by ViktorHofer** [comment]
I retried the build in that PR. Click on "Show previous attempts" under "Attempt 2".

**Comment by jkotas** [comment]
This is not actionable without a dump.

**Comment by ViktorHofer** [comment]
the log says "core dumped" but I have no idea where that dump would be found.

**Comment by trylek** [comment]
I think the problem is that this looks like a failure of the .NET CLI itself, the XUnit wrapper build as such seemingly finished fine. I'm not sure whether the test infra caters for this eventuality, probably not. In fact I think this is the first time I'm seeing this type of crash.

**Comment by jkotas** [comment]
Right, this is crash of .NET CLI itself. These crashed are relatively rare, but they do happen.

`kernel.core_pattern` kernel config controls where the dump is stored.

**Comment by trylek** [comment]
OK, I guess in such case the actionable aspect of this issue is to make sure we collect dumps from .NET CLI crashes and I'm going to rebrand it like that if you don't object.

**Comment by ViktorHofer** [comment]
Just spoke with MattGal about this and unfortunately we didn't get the dump as it wasn't uploaded. Dump collection unfortunately doesn't work in containers during building.
